### PR TITLE
Added initial keyword support

### PR DESCRIPTION
**Pull request by alancutter**

### CODE REVIEW

File: web-animations.js [hunk 1]
```diff
@@ -3277,7 +3280,7 @@ var propertyTypes = {
   'opacity': numberType,
   'outlineColor': colorType,
   // TODO: not clear why this is an integer in the transitions spec
```
Comment (dstoc) [inline web-animations.js]: Remove the TODO?

**Comment by dstoc** [comment]
Is the generator still not working? https://github.com/web-animations/web-animations-js/pull/337 was merged yesterday.

**Comment by alancutter** [comment]
The generator is has a console error when I click generate.

Uncaught TypeError: Cannot call method 'setTime' of undefined
test-generator.html:130

**Comment by mithro** [comment]
I this all good now?

**Comment by alancutter** [comment]
Still working out issues that have come up when testing CSS properties that haven't been tested before.
These are the current trouble makers:
backgroundPosition
borderBottomWidth
borderLeftWidth
borderRightWidth
borderTopWidth
outlineWidth
outlineOffset

**Comment by alancutter** [comment]
Generator and testharness are all good for this test now. Changed from manual test to regular test.

**Comment by shans** [comment]
LGTM. Please merge once you get the Travis build passing

**Comment by alancutter** [comment]
This is failing now due to certain keywords not being supported (letterSpacing: normal). I'm thinking keyword expansion (including initial) should be pulled out into its own stage in fromCssValue().

**Pull request closed without merging**